docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md

```diff
@@ -0,0 +1,430 @@
+
+* **ExpressRoute circuit**. A layer 2 or layer 3 circuit supplied by the connectivity provider that joins the on-premises network with Azure through the edge routers. The circuit uses the hardware infrastructure managed by the connectivity provider.
+
+> [!NOTE]
+> Our sample reference architecture uses a VPN connection, instead of an ExpressRoute circuit.
+
+* **ExpressRoute virtual network gateway**. The ExpressRoute virtual network gateway enables the VNet to connect to the ExpressRoute circuit used for connectivity with your on-premises network.
+
+* **Virtual network gateway**. The  virtual network gateway enables the VNet to connect to the VPN appliance in the on-premises network, or routers used for an ExpressRoute circuit. For more information, see [Connect an on-premises network to a Microsoft Azure virtual network][connect-to-an-Azure-vnet].
+
+* **VPN connection**. The connection has properties that specify the connection type (IPSec) and the key shared with the on-premises VPN appliance to encrypt traffic.
+
+* **Hub virtual network (VNet)**. Azure VNet used as a hub in a hub-spoke topology. The hub is used as a central point of connectivity to your on-premises network, and a place to host services that can be consumed by different workloads hosted in spoke VNets.
+
+    * **Gateway subnet**. The virtual network gateways are held in the same subnet.
+
+    * **Shared services subnet**. A subnet in the hub VNet used to host services that can be shared among all spokes, such as DNS, AD DS, among others.
+
+* **VNet peering**. You can connect two VNets in the same Azure region using a peering connection. Once peered, paired VNets exchange traffic by using the Azure backbone, without the need of a router. [Virtual networking peering][vnet-peering] connections are non-transitive, low latency connections between VNets.
+
+* **Spoke VNet**. Azure VNet used as a spoke in a hub-spoke topology. Spokes can be used to isolate wrokloads in their VNets, that can be managed separatly from other spokes. Each workload might include multiple tiers, with multiple subnets connected through Azure load balancers. For more information about the application infrastructure, see [Running Windows VM workloads][windows-vm-ra] and [Running Linux VM workloads][linux-vm-ra].
+
+## Recommendations
+
+The following recommendations apply for most scenarios. Follow these recommendations unless you have a specific requirement that overrides them.
+
+### VNet and GatewaySubnet
+
+Create the ExpressRoute virtual network gateway and the VPN virtual network gateway in the same VNet. This means that they should share the same subnet named *GatewaySubnet*.
```

Comment:
Does this RA derive from the ER-with-VPN-failover RA? Can we point to that for information about setting up the gateway, to avoid repeating info here?